.gitignore

```diff
@@ -101,3 +101,6 @@ typings/
 
 # TernJS port file
 .tern-port
+
+#github token
+/server/qa/config.js
```

Comment:
We should generate an API token for the organization `catlike-reactflexes` and all use the same token